frontend/src/state/restInterfaces.ts

```diff
@@ -626,7 +626,8 @@ export interface SchemaDetails {
     schemaId: number;
     version: number;
     compatibility: string;
-    schema: Schema;
+    type: string;
```

Comment:
Let's use an enum here, so that we can compare against it?

Reply:
done